NuGet.config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <!-- <add key="local" value="artifacts" /> -->
```

Comment:
This is used when building the DotNet.csproj if there are custom build bits, such as right now with the PR for enabling the profiler in Android.